Makefile

```diff
@@ -32,9 +32,6 @@ lint:
 test:
 	poetry run pytest tests/unit_tests
 
-tests:
```

Comment:
@hwchase17 -- can move this to another PR but not sure why there is both `make test and make tests`

Reply:
someone added this because its more canonical, dont see harm in having both

Reply:
reverted